services/pong/src/ping-store.js

```diff
@@ -0,0 +1,60 @@
+import { stats } from './messages';
+
+const time = () => Date.now() / 1000;
+
+/** Stores ping rates. */
+export default class PingStore {
+  /**
+   * Create a new PingStore.
+   *
+   * @param {Object[]} services
+   * @param {string}   services[].name
+   * @param {string}   services[].host
+   * @param {number}   services[].port
+   */
+  constructor(services) {
+    this._ping = stats.PingTimes.create({
+      time: time(),
+      // Get the services in the correct format and sort by name.
+      list: services.map(this._parseService.bind(this))
+        .sort((a, b) => a.name > b.name)
+    });
+  }
+
+  /**
+   * Feed new service ping to the store.
+   *
+   * @param {string}  name
+   * @param {boolean} online
+   * @param {number}  ms
+   */
```

Comment:
`ms`? Oh, milliseconds...

Reply:
`ms` is what it's called in the proto file so I kept it like that there.